forge/forge.js

```diff
@@ -75,6 +75,20 @@ module.exports = async (options = {}) => {
         trustProxy: true,
         logger: loggerConfig
     })
+
+    if (runtimeConfig.telemetry.backend?.prometheus?.enabled) {
+        const metricsPlugin = require('fastify-metrics')
+        await server.register(metricsPlugin, { endpoint: '/metrics' })
+    }
+
+    if (runtimeConfig.telemetry.backend?.sentry?.dsn) {
+        server.register(require('@immobiliarelabs/fastify-sentry'), {
+            dsn: runtimeConfig.telemetry.backend.sentry.dsn,
+            environment: process.env.NODE_ENV,
+            release: `flowforge@${runtimeConfig.version}`
+        })
+    }
```

Comment:
Did you mean to include the `sentry` stuff in here? That has more changes than just this and the dependency is not in the `package.json` included here.

```suggestion
```

Reply:
We are validating the sentry work in staging before deciding whether to adopt in production as well (which is why it didn't get backported automatically).

Given this is guarded on `runtimeConfig.telemetry.backend?.sentry?.dsn` - which should not get set in production then this should be okay as is.

Reply:
Ah - but the linting won't like the `require` if not also in package.json.

Let me discuss with @Pezmc if there's any reasons at this point not to get sentry setup on production as well.

Reply:
@hardillb Didn't spot this in the diff, usually cherry-pick doesn't show one. As the sentry changes created a conflict, I figured it was already in the maintenance branch.

I can manually remove it for now, to make it easier for this PR to go through without any additional scope inflation.